### DISCUSSION TITLE
while creating a new Repository   i am asked if i want to get / make a new repository-template

### DISCUSSION THREAD

**Discussion by fsj-digital** [community/community#33805] [Repositories]
while creating a new Repository 

i am asked if i want to get / make a new repository-template

repository template
Start your repository with a template repository's contents

![image](https://user-images.githubusercontent.com/32641183/191708237-781ed4b5-b96a-48a1-94b1-602dc5c4e7fc.png)

well what does this mean !?  Can i skip this - or do i get some preconfigured templates for a page!?

**Comment by Matteo-it** [comment]
Selecting a template your repo will be not empty, but will start with a content, based on the template you select in this panel! (You can anyway skip this and not use a template)

**Comment by fsj-digital** [comment]
many thanks dear Matteo-it. This helps alot. I played with it and now i think i get familiar

**Comment by fsj-digital** [comment]
btw. whats this - is this page created with the jekill style!`?

![image](https://user-images.githubusercontent.com/32641183/191719687-037f003e-e21f-4d82-b87f-1362911dabc4.png)

**Reply by fsj-digital** [reply]
btw - is this also written in Jekyll !? 

https://carolstran.github.io/cv/
i guess that this is a jekill template

**Reply by fsj-digital** [reply]
Hello dear all, 


i like the Theme Orderedlist - how to apply on Githubpage!?

saw it on the cv-page of  https://carolstran.github.io/cv/

https://github.com/orderedlist

look forward to hear from you

 regards
